public/templates/prSignUp.html

```diff
@@ -107,7 +107,7 @@
       <div class="tab-content">
         <!-- ***** Sign Up Form Start ***** -->
         <div id="signup">
-          <h1 id="signupText">Sign Up for Free</h1>
+          <h1 id="signup-text">Sign Up for Free</h1>
```

Comment:
Seems like we need to have a convention here dashed or camel case. The line directly above it is inconsistent with this line `id="signup"` is neither camel or snake case. 

Maybe let's standardize this:
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Which says use-dashes as you have done. Could you fix this in this file?

Reply:
Yep!